src/site/components/events_list.drupal.liquid

```diff
@@ -6,46 +6,49 @@
   {% endif %}
   {% assign sortedEventsArray = eventsArray | eventSorter %}
   {% for event in sortedEventsArray %}
-  {% if event.title.length %}
-  {% assign count = count | plus:1 %}
-  <div class="vads-u-margin-bottom--2">
-    <h4><a href="{{event.entityUrl.path}}">{{event.title}}</a></h4>
-    <p>{{event.fieldDescription}}</p>
-    <div class="usa-grid usa-grid-full">
-      {% if event.fieldDate.value %}
-      <div class="usa-width-one-sixth">
-        <strong>When:</strong>
+    {% if event.title.length %}
+      {% assign count = count | plus:1 %}
+      <div class="vads-u-margin-bottom--2">
+        <h4>
+          <a href="{{event.entityUrl.path}}">{{event.title}}</a>
+        </h4>
+        <p>{{event.fieldDescription}}</p>
+        <div class="usa-grid usa-grid-full">
+          {% if event.fieldDate.value %}
+            <div class="usa-width-one-sixth">
+              <strong>When:</strong>
+            </div>
+            <div class="usa-width-five-sixths">
+              <span>{{ event.fieldDate.value | humanizeDate }}</span><br>
+              <span>{{ event.fieldDate.value | humanizeTime }}
+                –
+                {{ event.fieldDate.endValue | humanizeTime }}</span>
```

Comment:
@ethanteague Does there need to be any logic to handle if there is no endValue, or does it always have an endValue?

Reply:
Not sure if there will always be an end value, but liquid (and filters) won't fail if this value doesn't exist - it just returns a falsy value, @swirtSJW

Reply:
Good call - adding some checks, @swirtSJW